src/blueprints/tv2/timeline-object-factories/interfaces/tv2-graphics-split-screen-timeline-object-factory.ts

```diff
@@ -1,7 +1,8 @@
 import { Tv2BlueprintTimelineObject } from '../../value-objects/tv2-metadata'
+import { GraphicsSetup, SplitScreenConfiguration } from '../../value-objects/tv2-show-style-blueprint-configuration'
 
 export interface Tv2GraphicsSplitScreenTimelineObjectFactory {
   createSplitScreenKeyTimelineObject(keyFilePath: string): Tv2BlueprintTimelineObject
   createSplitScreenFrameTimelineObject(frameFilePath: string): Tv2BlueprintTimelineObject
-  createSplitScreenLocatorTimelineObject(): Tv2BlueprintTimelineObject
+  createSplitScreenLocatorTimelineObject(graphicsSetup: GraphicsSetup, splitScreenConfiguration: SplitScreenConfiguration, locatorLabels?: string[]): Tv2BlueprintTimelineObject
```

Comment:
Since `locatorLabels` is a string array, then an empty array would probably suffice to describe a split screen without locator.

Reply:
We could, but then we are forced to add an empty array all the places where we create SplitScreens and we don't have any locators:
```
this.graphicsSplitScreenTimelineObjectFactory.createSplitScreenLocatorTimelineObject(blueprintConfiguration.showStyle.selectedGraphicsSetup, splitScreenConfiguration, [])
```
But I guess we should just not create the locator object in those cases?

Reply:
By not creating the timeline object, could the current split screen get locators from a lookahead?

Reply:
Yes, it would take it from lookahead.
So if SplitScreen A is on Air and don't have locators, but SplitScreen B is next and have locators. The locators from B would be applied and showed for A if we don't create the empty object.